doc/glossary.rst

```diff
@@ -1278,7 +1278,7 @@ Methods
         NOTE: Using ``partial_fit`` after ``fit`` results in undefined behavior.
 
     ``predict``
-        Makes a prediction for each sample, usually only taking :term:`X` as
+        Predicts each sample, usually only taking :term:`X` as
```

Comment:
the former sentence is more accurate

"to predict a sample" isn't common usage